src/codeu/chat/client/commandline/Chat.java

```diff
@@ -16,6 +16,7 @@
 
 import java.util.List;
 import java.util.Stack;
+import java.util.*;
```

Comment:
ohh, interesting. I thought I was being more efficient by just importing it all in one line! XD Thanks for the advice!